objects/src/accounts/account_id.rs

```diff
@@ -2,6 +2,88 @@ use super::{Account, AccountError, Digest, Felt, Hasher, StarkField, ToString, V
 use core::fmt;
 use crypto::FieldElement;
 
+#[cfg(feature = "log")]
+mod log {
```

Comment:
minor nit: I'd probably move this module to the end of the file.

Reply:
moved down